testsuite/testcases/src/validator_reboot_stress_test.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{LoadDestination, NetworkLoadTest};
+use forge::{NetworkContext, NetworkTest, Result, Swarm, Test};
+use rand::seq::SliceRandom;
+use rand::thread_rng;
+use std::time::Duration;
+use tokio::runtime::Runtime;
+use tokio::time::Instant;
+
+pub struct ValidatorRebootStressTest;
+
+impl Test for ValidatorRebootStressTest {
+    fn name(&self) -> &'static str {
+        "validator reboot stress test"
+    }
+}
+
+impl NetworkLoadTest for ValidatorRebootStressTest {
+    fn setup(&self, _swarm: &mut dyn Swarm) -> Result<LoadDestination> {
+        Ok(LoadDestination::AllFullnodes)
+    }
+
+    fn test(&self, swarm: &mut dyn Swarm, duration: Duration) -> Result<()> {
+        let start = Instant::now();
+        let runtime = Runtime::new().unwrap();
+
+        let all_validators = swarm.validators().map(|v| v.peer_id()).collect::<Vec<_>>();
+
+        let mut rng = thread_rng();
+
+        while start.elapsed() < duration {
+            let validator_to_reboot = swarm
+                .validator_mut(*all_validators.choose(&mut rng).unwrap())
+                .unwrap();
+            runtime.block_on(async { validator_to_reboot.stop().await })?;
+            runtime.block_on(async { validator_to_reboot.start().await })?;
+            std::thread::sleep(Duration::from_secs(15));
```

Comment:
out of curiousity, why sleep 15s here and 5s for fullnodes?

Reply:
I suspect we will lose quorum here. I've tried a script to restart 2 validators (out of 10) every 2 minutes on alden net, and the network sometimes cannot move forward.

Reply:
I will change it to 15s for FNs as well. 5s is just too frequent.